data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -145,7 +145,7 @@
 
         [message]
             speaker=Galim
-            message= _ "We have cleared the connecting tunnels to here and to the old keep as well as other parts o’ Knalga. It is slow and dangerous work, mostly due to the trolls. But it ha’ been made easier by Malifor’s ending."
+            message= _ "We have cleared the connecting tunnels from here to the old keep as well as other parts o’ Knalga. It is slow and dangerous work, mostly due to the trolls. But it ha’ been made easier by Malifor’s destruction."
```

Comment:
I like the change to "from".

"Destruction" is awkwardly ambiguous, it can be read both as "it's been made easier now Malifor is out of the way" and "it's been made easier because Malifor killed a lot of trolls too".